packages/flutter/lib/src/services/text_input.dart

```diff
@@ -585,6 +585,18 @@ abstract class TextSelectionDelegate {
   /// Brings the provided [TextPosition] into the visible area of the text
   /// input.
   void bringIntoView(TextPosition position);
+
+  /// Whether cut is enabled.
+  bool get cutEnabled;
```

Comment:
This is a breaking change.  I think it's entirely worth making, but we should probably send an announcement out to get feedback.

Reply:
Alternatively, we make these into a `mixin` that we mix in with our implementation.

Reply:
Nevermind abou tthe mixin - it's really probably best to just break here.

Reply:
+1 on just announcing this as a breaking change.

Is this class mainly extended or implemented as Interface? If it is the first we could reduce the breaking change impact by defaulting the impl to allow all.

Reply:
I only saw it in two places, one in editable_text that is implemented, one in the test that is extended, I think it is good to have default implementation. I will add that

Reply:
fixed